packages/modules/fulfillment/src/models/fulfillment-provider.ts

```diff
@@ -9,6 +9,7 @@ import {
 
 @Entity()
 export default class FulfillmentProvider {
+  @Searchable()
```

Comment:
Does adding this require modifying the migration?

Reply:
No it doesn't, this is all you need